cmake/external/composable_kernel.cmake

```diff
@@ -1,5 +1,5 @@
 set(composable_kernel_URL https://github.com/ROCmSoftwarePlatform/composable_kernel.git)
-set(composable_kernel_TAG bef0cb20dba0d9b315df46899310478a81c21852) # 2023-02-16 11:54:08 -0800
+set(composable_kernel_TAG ed3a2e52265e11daa366f47b082141a652b67c58) # 2023-04-10 21:02:17 +0800
```

Comment:
Need change cmake/deps.txt and tools/ci_build/github/azure-pipelines/templates/download-deps.yml for updated package?

Reply:
We don't need to do this for Composable Kernel.

Reply:
We did that for cutlass. Maybe it is because there is no packaging pipeline for Rocm EP?

Reply:
@tianleiwe Maybe we will do it in latter PR. The CK is actively updated, and they don't have a offical stable release at the moment.